todo_app/data/trello_items.py

```diff
@@ -1,68 +1,70 @@
-#trello_items.py
-
 import requests
 
-def fetch_todo_items(list_id, api_key, api_token):
-    url = f"https://api.trello.com/1/lists/{list_id}/cards?key={api_key}&token={api_token}"
-    response = requests.get(url)
-    data = response.json()
+class Item:
```

Comment:
This is a great example of a class, _but_ it's not quite honest about what it is - this class isn't representing a single `Item` (or `ToDo`), it's representing a set of functions that are useful for talking to Trello so it's not quite the structure we're expecting. I'd suggest:
* Update the name for this class to be a bit more accurate, e.g. for me this is is most naturally a `TrelloService` class - it's a class that provides useful access to Trello
* Potentially add a separate Item class that captures the concept of a single Item:
```python
class Item:
    def __init__(self, id, name, status = 'To Do'):
        self.id = id
        self.name = name
        self.status = status

    @classmethod
    def from_trello_card(cls, card, list):
        return cls(card['id'], card['name'], list['name'])
```

That class can then be used in turn in your `fetch_todo_item` function:
```
        todo_items = []
        for item in data:
            todo_items.append(Item(item))
```

Reply:
(Note that if you do go for this, you'd need to tweak how you access those names on the frontend, e.g. with `item.name` instead of just `item`)

Reply:
Done